sdk/tools/azure-openrewrite/pom.xml

```diff
@@ -126,6 +147,181 @@
                 <artifactId>maven-pmd-plugin</artifactId>
                 <version>3.15.0</version>
             </plugin>
+            <!-- generate sources jar -->
```

Comment:
Let's chat about what is going on here - I'm not sure I understand.

Reply:
Because the CI pipeline that will be running this is 'java - tools', I added all the plugins that existed in `azure-sdk-build-tools` because the tools directory currently doesn't have a parent pom.

Reply:
Hmm, should we just have this project just use the parent POM? I don't think there would be a concern doing that if we want this package to have the same build process.

Reply:
@alzimmermsft are you aware of any substantial build differences between the client ci's and the tools ci?